models/models.go

```diff
@@ -313,20 +307,21 @@ func (m *RouteManager) Update(instanceId, domain, origin string, path string, in
 	route.DistId = *dist.Id
 
 	if domain != "" {
-		user, err := route.loadUser(m.db)
-		if err != nil {
-			return err
-		}
-
-		clients, err := m.getClients(&user, m.settings)
-		if err != nil {
-			return err
-		}
-
-		route.ChallengeJSON = []byte("")
-		if err := m.ensureChallenges(route, clients[acme.HTTP01], false); err != nil {
-			return err
-		}
+		return errors.New("the domain cannot be empty")
```

Comment:
this logic seems backwards - if `domain` is *not* an empty string, we're raising an error about it being empty?. Also, we should favor deleting code over commenting it.

Reply:
removed since it was wrong and not needed.